packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Primitives/PropertyWireInformation.cs

```diff
@@ -29,7 +29,7 @@ internal PropertyWireInformation(InputModelProperty inputModelProperty)
             SerializationFormat = CodeModelPlugin.Instance.TypeFactory.GetSerializationFormat(inputModelProperty.Type);
             IsRequired = inputModelProperty.IsRequired;
             IsReadOnly = inputModelProperty.IsReadOnly;
-            SerializedName = inputModelProperty.SerializedName.FirstCharToUpperCase();
```

Comment:
by definition, the serialized name should be exactly the serialized name we get from the emitter, we should never change it.
If we need to change it, it means this is not the "serialized name"